src/Courses/CourseDetailsTabs/StatisticsTab.tsx

```diff
@@ -137,6 +138,7 @@ export const StatisticsTab: React.FC<StatisticsTabProps> = ({ course, userId })
     const globalView = statisticsViewFromAllStatisticsViewFilter(view);
 
     useEffect(() => {
+        logger.info(`Stats tab: [useEffect] course.id ${course.id}, globalView ${globalView}, idFilter ${idFilter}, userId ${userId}, userType ${userType}`);
```

Comment:
We should consider making a PREFIX variable, just in case someone mistypes the prefix.

Reply:
Yeah I'm between two approaches here:
1. Add a tag variable like android logcat does
2. Create logging interfaces that add the tag for you
  * I wanted this to automatically happen however I ran into issues with the source map
  * I know typescript types are stripped but I wonder if you could get something like a toString and bind it to the logger